.github/workflows/msrv.yml

```
@@ -0,0 +1,36 @@
+name: Check MSRV
+
+on:
+  push:
+    branches: [next]
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+# Limits workflow concurrency to only the latest commit in the PR.
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  # Check MSRV (aka `rust-version`) in `Cargo.toml` is valid for workspace members
+  msrv:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: sudo apt-get update && sudo apt-get install -y jq
+      - uses: dtolnay/rust-toolchain@stable
```

nit: I think we can do
```suggestion
      - uses: dtolnay/rust-toolchain@stable
        with:
          targets: wasm32-unknown-unknown
```

and avoid the later step